scripts/post-changelog.mjs

```diff
@@ -1,17 +1,15 @@
 import { $, fetch, argv } from "zx";
 import dotenv from "dotenv-safe";
-import conventionalChangelog from "conventional-changelog";
-import { getPackages } from "@lerna/project";
-import slackify from "slackify-markdown";
+import slackifyMarkdown from "slackify-markdown";
 import { simpleGit } from "simple-git";
 import gitDiffParser from "gitdiff-parser";
 
 /* eslint-disable no-console */
 
-const CHANNEL = `orbit-react`;
-const COLOR_CORE = `#00A58E`;
-const PACKAGES = ["orbit-components", "orbit-tailwind-preset"];
+const CHANNEL = "orbit-react";
 const PACKAGE_PREFIX = "@kiwicom";
```

Comment:
It's used only once, maybe does not make sense to have a constant for that

better will be to adjust the function make it completely pure: 
```
function format(str, package_, prefix = "@kiwicom") { 
```